### ISSUE TITLE
torch.hub.load() crash when loading fairseq model

### ISSUE THREAD

**Issue by arnaudsm** [pytorch/fairseq#1467]
I'm trying to run [this basic code](https://pytorch.org/hub/pytorch_fairseq_roberta/) from the PyTorch documentation :
```
!pip install torch==1.3.0 regex requests

import torch
roberta = torch.hub.load('pytorch/fairseq', 'roberta.large')
roberta.eval()  # disable dropout (or leave in train mode to finetune)
```
And it crashes, saying `TypeError: Required argument 'file' (pos 1) not found`.
I've been trying with `force_reload=True` and other fairseq models, with no success..

Here's the entire stacktrace : 

```
TypeError                                 Traceback (most recent call last)
<ipython-input-47-3f3e0ab33e0e> in <module>
      2 
      3 import torch
----> 4 roberta = torch.hub.load('pytorch/fairseq', 'roberta.large', force_reload=True)
      5 roberta.eval()  # disable dropout (or leave in train mode to finetune)

/usr/local/lib/python3.6/dist-packages/torch/hub.py in load(github, model, *args, **kwargs)
    357     entry = _load_entry_from_hubconf(hub_module, model)
    358 
--> 359     model = entry(*args, **kwargs)
    360 
    361     sys.path.remove(repo_dir)

~/.cache/torch/hub/pytorch_fairseq_master/fairseq/models/roberta/model.py in from_pretrained(cls, model_name_or_path, checkpoint_file, data_name_or_path, bpe, **kwargs)
    142             bpe=bpe,
    143             load_checkpoint_heads=True,
--> 144             **kwargs,
    145         )
    146         return RobertaHubInterface(x['args'], x['task'], x['models'][0])

~/.cache/torch/hub/pytorch_fairseq_master/fairseq/hub_utils.py in from_pretrained(model_name_or_path, checkpoint_file, data_name_or_path, archive_map, **kwargs)
     66     models, args, task = checkpoint_utils.load_model_ensemble_and_task(
     67         [os.path.join(model_path, cpt) for cpt in checkpoint_file.split(':')],
---> 68         arg_overrides=kwargs,
     69     )
     70 

~/.cache/torch/hub/pytorch_fairseq_master/fairseq/checkpoint_utils.py in load_model_ensemble_and_task(filenames, arg_overrides, task)
    176         if not os.path.exists(filename):
    177             raise IOError("Model file not found: {}".format(filename))
--> 178         state = load_checkpoint_to_cpu(filename, arg_overrides)
    179 
    180         args = state["args"]

~/.cache/torch/hub/pytorch_fairseq_master/fairseq/checkpoint_utils.py in load_checkpoint_to_cpu(path, arg_overrides)
    143 def load_checkpoint_to_cpu(path, arg_overrides=None):
    144     """Loads a checkpoint to CPU (with upgrading for backward compatibility)."""
--> 145     with PathManager.open(path, "rb") as f:
    146         state = torch.load(
    147             f, map_location=lambda s, l: default_restore_location(s, "cpu")

~/.cache/torch/hub/pytorch_fairseq_master/fairseq/file_io.py in open(path, mode, buffering, encoding, errors, newline)
     49             encoding=encoding,
     50             errors=errors,
---> 51             newline=newline,
     52         )
     53 

TypeError: Required argument 'file' (pos 1) not found
```
Environnement : Docker `tensorflow/tensorflow:latest-gpu-py3-jupyter`

Thanks a lot for your help.

**Comment by arnaudsm** [comment]
I was able to reproduce the problem on Google Colab : https://colab.research.google.com/drive/1q5bKlZHKGjPD1P0AU6DhrFxeW4TZYZ0g

**Comment by myleott** [comment]
It broke in 51d2fe2503b3f6cd3e07689edcfa160d27a829a5. I have a fix will merge shortly.

**Comment by myleott** [comment]
This is fixed: af3050285d646125e7c76a245188b23a0b043185

**Comment by arnaudsm** [comment]
Thanks a lot for the quick fix !
